style.css

```diff
@@ -0,0 +1,99 @@
+body {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.wrapper {
+  background-image: url(/images/header-background-mobile.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  min-height: 720px;
+  border-bottom-left-radius: 100px;
+}
+
+ul {
+  padding: 0;
+}
+
+li {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 78px;
+  margin: 0 24px 24px 24px;
+}
+
+.logo {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  margin-top: 58px;
+}
+
+.hamburger {
+  margin-top: 58px;
+}
+
+.main {
+  height: 600px;
+  background-image: url(/images/header-shapes\ mobile.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: absolute;
+  top: 104px;
+}
+
+h1 {
+  margin: 0 24px 0 24px;
+  padding-top: 96px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;

```

Comment:
- [ ] I can see that your main header overlap with the background image please fix that and try to stick as much as possible to the given Figma design.
[hint] you can do it by adjusting your header width.

| Figma Design  | Your Design |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/68796232/164248292-71f3cc51-cbc5-4eb3-8f90-f48e21fdd30e.png)  | ![image](https://user-images.githubusercontent.com/68796232/164248325-7dfd472d-d24e-4203-be28-43e8fb73eacf.png)  |

Reply:
I think it is as a result of the font-family. Sans-serif instead of Poppins